content/applications/inventory_and_mrp/inventory/product_management/product_tracking/serial_numbers.rst

```diff
@@ -290,4 +346,11 @@ just one product per serial number.
    number.
 
 .. seealso::
-   :doc:`../product_tracking`
+   :doc:`/applications/inventory_and_mrp/inventory/product_management/product_tracking/reassign`
+
+.. tip::
+   In addition to using the :guilabel:`Lots/Serial Numbers dashboard`, there are several other
+   reporting templates that display serial number. Go to
+   :menuselection:`Inventory app --> Reporting`, where the :guilabel:`Locations`report,
+   :guilabel:`Moves History`report, and :guilabel:`Moves Analysis`report all include either
+   the :guilabel:`Lot/Serial Number` field or the ability to filter by serial number.
```

Comment:
All of these guilabels are missing a space between the closing backtick ` and the next work (:

I didn't directly make a suggestion, however, because I thought that maybe it would be better to do a doc link to these existing doc pages!